arviz/data/io_pymc3.py

```diff
@@ -1,29 +1,54 @@
 """PyMC3-specific conversion code."""
 import logging
 from typing import Dict, List, Any, Optional, TYPE_CHECKING
+from types import ModuleType
 
 import numpy as np
 import xarray as xr
 from .. import utils
-from .inference_data import InferenceData
+from .inference_data import InferenceData, concat
 from .base import requires, dict_to_dataset, generate_dims_coords, make_attrs
 
 if TYPE_CHECKING:
     import pymc3 as pm
+    from pymc3 import MultiTrace, Model  # pylint: disable=invalid-name
+    import theano
+    from typing import Set  # pylint: disable=ungrouped-imports
+else:
+    MultiTrace = Any  # pylint: disable=invalid-name
+    Model = Any  # pylint: disable=invalid-name
```

Comment:
I was just thinking, once we start adding type hinting to ArviZ, the use of `Any` may become quite common, maybe it is worth it to include `Any` as a good name in pylintrc

Reply:
Actually, it was `Model` and `MultiTrace` that pylint didn't like, because they look like class names not variable names -- which is what they are!  I'm pretty sure the `Any` was fine.

Reply:
Ohhh, yeah, that makes much more sense